src/lib/web-component/filter-option/filter-option.css

```diff
@@ -2,63 +2,69 @@
   box-sizing: border-box;
 }
 
+.filter-container {
+  width: auto;
+}
+
 .filter-checkbox {
   display: block;
   position: relative;
-  padding-left: 30px;
+  padding: 8px 5px 0 25px;
   cursor: pointer;
-  font-size: 18px;
-  margin: 10px 0;
+  font-size: 15px;
   -moz-user-select: none;
   -webkit-user-select: none;
 }
+
 .filter-checkbox input {
   position: absolute;
   z-index: -1;
   opacity: 0;
 }
+
 .select-indicator {
   position: absolute;
-  top: 0;
-  left: 0;
-  height: 20px;
-  width: 20px;
-  background: #e6e6e6;
+  top: 10px;
+  left: 5px;
+  height: 15px;
+  width: 15px;
+  background: #ededf0;
 }
 
 .filter-checkbox:hover input ~ .select-indicator,
 .filter-checkbox input:focus ~ .select-indicator {
   background: #ccc;
 }
+
 .filter-checkbox input:checked ~ .select-indicator {
-  background: #2aa1c0;
+  background: #00c8d7;
 }
+
 .filter-checkbox:hover input:not([disabled]):checked ~ .select-indicator,
 .filter-checkbox input:checked:focus ~ .select-indicator {
-  background: #0e647d;
-}
-.filter-checkbox input:disabled ~ .select-indicator {
-  background: #e6e6e6;
-  opacity: 0.6;
-  pointer-events: none;
+  background: #008ea4;
 }
+
```

Comment:
@atiqueahmedziad To make it easier to change the tweak the colors (at the moment to better match the color being used in the devtools panels, but also to make it possible to easily switch between themes, e.g. light and dark themes) it would be better to use CSS variables for the photon colors (which would also make it way easier to recognize which photon color a certain element is using) and also some CSS variables to separate the "kind of usage that a certain color is used for" from the actual photon color we are going to currently use for that.

I inspected the devtools panels (using the Browser Toolbox as I did suggest in our last meeting and also by looking for those variables on searchfox.org) and looked for the colors being used in the devtools panels and then I tried to quickly add some CSS variables on top of the changes in this PR without changing too much or the rest of the CSS rules.

Here is the comparison screenshot of the resulting activitylog page and the devtools panel:

![image](https://user-images.githubusercontent.com/11484/93377862-e5d6e680-f85b-11ea-85a3-81c47658bb56.png)

There are some more tweaks that may be reasonable to apply to match the devtools panel look and feel even more (where/when it makes sense), but this looks already as a good start from my perspective.

It is ok if you prefer to merge this pull request as is (after handling the last round of review comments from Rob) and make this changes into a separate pull request.

(as a side note, we should also do something similar for the colors used in the browserAction popup).

Here is the diff of the changes I applied locally:

```diff
diff --git a/src/activitylog/activitylog.css b/src/activitylog/activitylog.css
index bfc5bf6..bfbf619 100644
--- a/src/activitylog/activitylog.css
+++ b/src/activitylog/activitylog.css
@@ -54,6 +54,7 @@ input[type='file'] {
   grid-column-gap: 10px;
   align-items: center;
   font-size: 14px;
+  border-bottom: 1px solid var(--table-splitter-color);
 }
 
 .options-dropdown {
@@ -152,8 +153,9 @@ input[type='file'] {
   height: 30px;
   align-items: center;
   padding: 0 15px;
-  background: #00c8d7;
-  color: #ffffff;
+  background: var(--table-header-background-color);
+  color: var(--text-color);
+  border-top: 1px solid var(--table-splitter-color);
 }
 
 .logs-counter {
diff --git a/src/activitylog/activitylog.html b/src/activitylog/activitylog.html
index 3e42c08..4463ed9 100644
--- a/src/activitylog/activitylog.html
+++ b/src/activitylog/activitylog.html
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Extension Activity Monitor</title>
+    <link rel="stylesheet" href="/colors.css" />
     <link rel="stylesheet" href="./activitylog.css" />
     <script src="activitylog.js" type="module"></script>
     <script
diff --git a/src/colors.css b/src/colors.css
new file mode 100644
index 0000000..3e377fa
--- /dev/null
+++ b/src/colors.css
@@ -0,0 +1,85 @@
+:root {
+  /*
+   * Photon Colors CSS Variables v3.3.2
+   * - Colors are taken from https://github.com/FirefoxUX/photon-colors/blob/master/photon-colors.css
+   *   by the Firefox DevTools and available in mozilla-central as devtools/client/themes/variables.css.
+   * - Plus a few unofficial colors: a few intermediary values (e.g. Blue 45),
+   *   and lighter variants for the dark theme (e.g. Red 20, Red 40).
+   */
+  --magenta-50: #ff1ad9;
+  --magenta-65: #dd00a9;
+  --magenta-70: #b5007f;
+
+  --purple-50: #9400ff;
+  --purple-60: #8000d7;
+
+  --blue-30: #75baff;
+  --blue-40: #45a1ff;
+  --blue-50: #0a84ff;
+  --blue-50-a30: rgba(10, 132, 255, 0.3);
+  --blue-55: #0074e8;
+  --blue-60: #0060df;
+  --blue-70: #003eaa;
+  --blue-80: #002275;
+
+  --teal-60: #00c8d7;
+  --teal-70: #008ea4;
+
+  --green-50: #30e60b;
+  --green-60: #12bc00;
+  --green-70: #058b00;
+
+  --yellow-50: #ffe900;
+  --yellow-60: #d7b600;
+  --yellow-65: #be9b00;
+  --yellow-70: #a47f00;
+  --yellow-80: #715100;
+
+  --red-20: #ffb3d2;
+  --red-40: #ff3b6b;
+  --red-50: #ff0039;
+  --red-60: #d70022;
+  --red-70: #a4000f;
+
+  --grey-10: #f9f9fa;
+  --grey-10-a15: rgba(249, 249, 250, 0.15);
+  --grey-10-a20: rgba(249, 249, 250, 0.2);
+  --grey-10-a25: rgba(249, 249, 250, 0.25);
+  --grey-10-a30: rgba(249, 249, 250, 0.3);
+  --grey-20: #ededf0;
+  --grey-25: #e0e0e2;
+  --grey-30: #d7d7db;
+  --grey-35: #c3c3c6;
+  --grey-40: #b1b1b3;
+  --grey-43: #a4a4a4;
+  --grey-45: #939395;
+  --grey-50: #737373;
+  --grey-55: #5c5c5f;
+  --grey-60: #4a4a4f;
+  --grey-70: #38383d;
+  --grey-80: #2a2a2e;
+  --grey-85: #1b1b1d;
+  --grey-90: #0c0c0d;
+  --grey-90-a05: rgba(12, 12, 13, 0.05);
+  --grey-90-a10: rgba(12, 12, 13, 0.1);
+  --grey-90-a15: rgba(12, 12, 13, 0.15);
+  --grey-90-a20: rgba(12, 12, 13, 0.2);
+  --grey-90-a30: rgba(12, 12, 13, 0.3);
+}
+
+/* Components theme CSS variables */
+:root {
+  --text-color: var(--grey-70);
+
+  --table-header-background-color: var(--grey-10);
+  --table-splitter-color: var(--grey-25);
+  --table-odd-rows: var(--grey-90-a05);
+
+  --selection-background: var(--blue-55);
+  --selection-color: white;
+
+  --checkbox-color: var(--blue-40);
+  --checkbox-focused-color: var(--blue-55);
+}
+
+
diff --git a/src/lib/web-component/filter-option/filter-option.css b/src/lib/web-component/filter-option/filter-option.css
index ba2b49f..1bb1209 100644
--- a/src/lib/web-component/filter-option/filter-option.css
+++ b/src/lib/web-component/filter-option/filter-option.css
@@ -37,12 +37,12 @@
 }
 
 .filter-checkbox input:checked ~ .select-indicator {
-  background: #00c8d7;
+  background: var(--checkbox-color)
 }
 
 .filter-checkbox:hover input:not([disabled]):checked ~ .select-indicator,
 .filter-checkbox input:checked:focus ~ .select-indicator {
-  background: #008ea4;
+  background: var(--checkbox-focused-color)
 }
 
 .select-indicator:after {
diff --git a/src/lib/web-component/log-view/log-view.css b/src/lib/web-component/log-view/log-view.css
index 74791ed..a6ee517 100644
--- a/src/lib/web-component/log-view/log-view.css
+++ b/src/lib/web-component/log-view/log-view.css
@@ -53,8 +53,8 @@ table {
 table thead {
   position: sticky;
   top: 0;
-  background: #00c8d7;
-  color: #ffffff;
+  background: var(--table-header-background-color);
+  color: var(--text-color);
 }
 
 table thead th {
@@ -62,6 +62,7 @@ table thead th {
   font-size: 16px;
   position: relative;
   text-align: left;
+  border: 1px solid var(--table-splitter-color);
 }
 
 table tbody {
@@ -71,7 +72,7 @@ table tbody {
 table thead th:not(:first-child)::after {
   content: '';
   display: block;
-  border: 1px solid #d7d7db;
+  border-right: 1px solid var(--table-splitter-color);
   position: absolute;
   top: 0;
   bottom: 0;
@@ -79,7 +80,7 @@ table thead th:not(:first-child)::after {
 }
 
 table tbody tr:nth-of-type(odd) {
-  background: #ededf0;
+  background: var(--table-odd-rows);
 }
 
 table tbody tr {
@@ -92,15 +93,16 @@ table td {
 }
 
 table td:not(:first-child) {
-  border-left: 1px solid #d7d7db;
+  border-left: 1px solid var(--table-splitter-color);
 }
 
 table td:not(:last-child) {
-  border-right: 1px solid #d7d7db;
+  border-right: 1px solid var(--table-splitter-color);
 }
 
 .row-highlight {
-  background-color: #96eaf0 !important;
+  background-color: var(--selection-background) !important;
+  color: var(--selection-color);
 }
 
 .table-empty-label {
```

Reply:
Thanks a lot @rpl for explaining and for putting the diff. I have applied these styles.